src/context/data-context.jsx

```diff
@@ -0,0 +1,48 @@
+import { useReducer, createContext, useContext } from "react";
+
+const DataContext = createContext(null);
```

Comment:
you don't need to pass null explicitly unless you are certain that it's a good practice.